Library/ViewModels/AddNewCardViewModel.swift

```diff
@@ -35,6 +35,7 @@ public protocol AddNewCardViewModelOutputs {
   var creditCardValidationErrorContainerHidden: Signal<Bool, Never> { get }
   var cardholderNameBecomeFirstResponder: Signal<Void, Never> { get }
   var dismissKeyboard: Signal<Void, Never> { get }
+  var newCardAdded: Signal<GraphUserCreditCard.CreditCard, Never> { get }
```

Comment:
In order for us to make this more readable, maybe we can define a typealias..

Currently there are 2 `CreditCard` structs
* GraphUserCreditCard.CreditCard
* Query.User.CreditCard

How about we declare typealias `public typealias GraphCreditCard = GraphUserCreditCard.CreditCard`?

Reply:
Sounds good, let's not do that as part of this PR though 👍